solara/website/pages/documentation/examples/utilities/countdown_timer.py

```diff
@@ -41,24 +41,26 @@ def run_timer():
 
     solara.use_thread(run_timer, dependencies=[duration, running])
 
-    with solara.VBox() as main:
-        if not running:
-            if duration < 1:
-                solara.Error("Duration must be at least 1 second")
-            else:
-                solara.Markdown(f"# Timer set to {seconds} seconds")
+    # with solara.VBox() as main:
+    solara.Column()
```

Comment:
This is not needed, if you have multiple children, you will get this automatically, so this can be removed.